AdminNotificationManagerPlugin.inc.php

```diff
@@ -146,19 +148,31 @@ function getTemplatePath($inCore = false) {
 	 * @return array
 	 */
 	function _getAdminList() {
-		$user = null;
+		$arrayOfContexts = $this->_getContexts();
 		$users = array();
+		$userId = array();
+		$user = null;
+		$userFactory = new PKPUserService();
 
-		$roleDAO = DAORegistry::getDAO('RoleDAO');
-		$userDAO = $roleDAO->getUsersByRoleId(ROLE_ID_SITE_ADMIN);
+		foreach($arrayOfContexts as $context) {
+		$userGroupDAO = DAORegistry::getDAO('UserGroupDAO');
+		$userGroupDAOFactory = $userGroupDAO->getByRoleId($context,ROLE_ID_SITE_ADMIN);
 
-		if ($userDAO && $userDAO->getCount() > 0) {
-			while ($user = $userDAO->next()) {
-				$users[$user->getId()] = $user;
+		if ($userGroupDAOFactory && $userGroupDAOFactory->getCount() >= 1) {
+			while($group = $userGroupDAOFactory->next()) {
+				$groupId = $group->getId();
+				$userId[] = $groupId;     
 			}
 		}
-		return $users;
+			$args = array('userGroupIds'=>$userId);
+			$listOfUsers= $userFactory->getMany($args);
+			foreach($listOfUsers as $user) {
+				$idValue = $user->getId();
+				$users[$idValue] = $user;   
+			}
 	}
+	return $users;
```

Comment:
This `return` and the closing brace to the `function` is outdented one level too much.